libopae/plugins/xfpga/metrics/metrics_utils.c

```diff
@@ -254,6 +254,9 @@ fpga_result enum_thermalmgmt_metrics(fpga_metric_vector *vector,
 
 	for (i = 0; i < pglob.gl_pathc; i++) {
 
+		if (!pglob.gl_pathv)
+			continue;
```

Comment:
Probably want break instead of continue.

Reply:
fixed